### PR TITLE
Add 1.5.29 post

### PR DESCRIPTION
**Pull request by Adam-**

### CODE REVIEW

File: src/_posts/2019-07-18-10-00-1.5.29-Release.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: '1.5.29 Release'
+description: 'Seed vault price checker, scoll zoom speed changer, and bug fixes'
```
Comment (Alexsuperfly) [inline src/_posts/2019-07-18-10-00-1.5.29-Release.md]:
```suggestion
description: 'Seed vault price checker, scroll zoom speed changer, and bug fixes'
```

File: src/_posts/2019-07-18-10-00-1.5.29-Release.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+author: Adam
+---
+
+The bank plugin will now also total up the value of all of the seeds in the seed
```
Comment (Alexsuperfly) [inline src/_posts/2019-07-18-10-00-1.5.29-Release.md]:
```suggestion
The bank plugin will now total up the value of all of the seeds in the seed
```

File: src/_posts/2019-07-18-10-00-1.5.29-Release.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+vault
+
+![seedvault](/img/blog/1.5.29-Release/seedvault.png)
+
+The camera zoom plugin now has a setting to configure the scroll zoom speed, and
+allows configuring the control key to reset zoom.
+
+The grounditems plugin can now also recolor the telegrab menu to according to the items value
```
Comment (Alexsuperfly) [inline src/_posts/2019-07-18-10-00-1.5.29-Release.md]:
```suggestion
The grounditems plugin can now recolor the telegrab menu to according to the items value
```

File: src/_posts/2019-07-18-10-00-1.5.29-Release.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+- Fix location of Fishing Platform elite clue stash unit
+- Fix HD prayer bar flick bar
+- Add Entrana river fishing spots to fishing plugin
+- Fix MTA telekinetic solver sometimes not updating after telegrabbing the
+  guardian
+- Fix several edge cases related to untradeables in the items kept on death
+  plugin
+- Add support for labeling ore and gems to the item identification plugin
+- Add menu swaps for Tob Quick-Enter and Enchant on Eluned
+- Add Rantz arrow collection has been added to the daily task plugin
```
Comment (Alexsuperfly) [inline src/_posts/2019-07-18-10-00-1.5.29-Release.md]:
```suggestion
- Add Rantz arrow collection to the daily task plugin
```

File: src/_posts/2019-07-18-10-00-1.5.29-Release.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+- Add Sarachnis respawn timer
+- Fix location of Fishing Platform elite clue stash unit
+- Fix HD prayer bar flick bar
+- Add Entrana river fishing spots to fishing plugin
+- Fix MTA telekinetic solver sometimes not updating after telegrabbing the
+  guardian
+- Fix several edge cases related to untradeables in the items kept on death
+  plugin
+- Add support for labeling ore and gems to the item identification plugin
+- Add menu swaps for Tob Quick-Enter and Enchant on Eluned
```
Comment (Nightfirecat) [inline src/_posts/2019-07-18-10-00-1.5.29-Release.md]: "ToB" is both more accurate and more recognizable than "Tob" imo

```suggestion
- Add menu swaps for ToB Quick-Enter and Enchant on Eluned
```

**Pull request merged**